docs/nuxt/getting-started.md

```diff
@@ -78,6 +78,13 @@ This service account file contains sensitive information and should **not be com
 echo service-account.json >> .gitignore
 ```
 
+:::tip
+During deployment to Google Cloud or Firebase, it is important that the .env file pointing to a local `service-account.json` is not deployed to Firebase. If you're deploying from a local machine and you forget to delete your `.env` file, you may get the following error in your Google Cloud function logs:
```

Comment:
Thanks for the PR but this information of _`.env` file should not be deployed` is not something that belongs in VueFire docs specifically as it's a general rule of thumb in deployments. Usually the tools that deploy ignore these files, you should open an issue in the deployment tool you use so it automatically ignores local env files